test/tests/phi_zero/tests

```diff
@@ -0,0 +1,22 @@
+[Tests]
+  [./phi_zero_linear]
+    type = 'RunApp'
+    input = 'simple_transient_diffusion.i'
+    expect_out = ' Solve Converged!'
+    issues = '#15204'
+    design = ' '
+    requirement = 'The system shall be able to produce consestent size _phi_zero and _grad_phi_zero objects'
+    method = 'dbg'
```

Comment:
You might as well remove this as well. I understand that your assertions won't be tested, but the problem should still run for all methods